functions-and-scope.js

```diff
@@ -14,6 +15,27 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan bijhouden?
 // Log het antwoord in de terminal.
 
+const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
+const grades2 = [6,4,5];
+const grades3 = [8,9,4,6,10]
+
+function cumLaudeGrades(arr) {
+    let counter = 0;
+    for (let i = 0; i < arr.length; i++) {
+        if (arr[i] > 7){
+            counter += 1;
```

Comment:
Dit kun je nog iets versimpelen
```suggestion
            counter++;
```